src/qnngds/devices.py

```diff
@@ -0,0 +1,263 @@
+""" 
+Devices module contain the basic devices that QNN uses in its circuits (e.g. ntron, htron etc).
+"""
+
+from __future__ import division, print_function, absolute_import
+from phidl import Device
+# from phidl import quickplot as qp
```

Comment:
I'd go through and double check and remove unused imports in the python source files